weblate/templates/trans/alert/duplicatelanguage.html

```diff
@@ -9,11 +9,11 @@
 
     <ul>
     {% if not analysis.monolingual %}
-      <li>{% trans "You might be using monolingual translation files as bilingual. In that case please configure the monolingual base language file." %} {% documentation_icon "formats" "bimono" %}</li>
+      <li>{% trans "You might be using a monolingual translation file as a bilingual one. If that is the case, please set the monolingual base language file origin." %} {% documentation_icon "formats" "bimono" %}</li>
```

Comment:
I still think the origin word is not really helpful here. It's about filling in https://docs.weblate.org/en/latest/admin/projects.html#monolingual-base-language-file in the component configuration.

```suggestion
      <li>{% trans "You might be using a monolingual translation file as a bilingual one. If that is the case, please set the monolingual base language file." %} {% documentation_icon "formats" "bimono" %}</li>
```

Reply:
Went with location. Anything to help the user understand the file is already there in need of pointing out.